com.unity.netcode.gameobjects/Runtime/Serialization/SerializationTypeTable.cs

```diff
@@ -0,0 +1,449 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// Registry for telling FastBufferWriter and FastBufferReader how to read types when passed to
+    /// WriteObject and ReadObject, as well as telling BytePacker and ByteUnpacker how to do it when passed to
+    /// WriteObjectPacked and ReadObjectPacked.
+    ///
+    /// These object-based serialization functions shouldn't be used if at all possible, but if they're required,
+    /// and you need to serialize a type that's not natively supported, you can register it with the dictionaries here:
+    ///
+    /// Serializers and Deserializers for FastBufferWriter and FasteBufferReader
+    /// SerializersPacked and DeserializersPacked for BytePacker and ByteUnpacker
+    /// </summary>
+    public static class SerializationTypeTable
+    {
+        public delegate void Serialize(ref FastBufferWriter writer, object value);
+        public delegate void Deserialize(ref FastBufferReader reader, out object value);
```

Comment:
```suggestion
        public delegate void SerializeHandler(ref FastBufferWriter writer, object value);
        public delegate void DeserializeHandler(ref FastBufferReader reader, out object value);
```

Reply:
Removed this whole file, since it was only used for object serialization, which I have now removed.